### PR TITLE
refacto stop_monitoring to use navitia_model and a timetable

### PR DESCRIPTION
**Pull request by antoine-de**
we generate the a csa like timetable (with all the sorted departures), this way we handle easily all the corner cases (validity pattern, pass midnight, ...)

I need to add some unit tests, but I think this can be done in another PR

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -42,7 +42,13 @@ fn main() {
     let sys = actix::System::new("transpo-rt");
     let params = Params::from_args();
     let bind = format!("{}:{}", &params.bind, &params.port);
-    let context = transpo_rt::server::make_context(&params.gtfs, &params.url);
+
+    let today = chrono::Local::today(); //TODO use the timezone's dataset ?
+    let period = transpo_rt::context::Period {
+        begin: today.naive_local(),
+        end: today.succ().naive_local(),
```
Comment (Tristramg) [inline src/main.rs]: Comme l’inégalité est stricte, on est que sur 24h, non ? donc si le serveur tourne 20h, ça devient limite comme utilisabilité, non ?

File: src/context.rs [hunk 2]
```diff
@@ -27,8 +42,75 @@ pub struct Context {
...
+    pub fn contains(&self, date: NaiveDate) -> bool {
+        self.begin <= date && date < self.end
+    }
+}
+
+// create a dt from a Date and a StopTime's time
+// Note: the time might be on the next day, for example "26:00:00"
+// is the next day at 2 in the morning
+fn create_dt(date: NaiveDate, time: navitia_model::objects::Time) -> NaiveDateTime {
+    let date = if time.hours() > 24 { date.succ() } else { date };
```
Comment (TeXitoi) [inline src/context.rs]: Je pense que c'est surtout la que ça marche pas avec le transsibérien.

Comment (antoine-de) [reply src/context.rs]: oui oui, ca ne fonctionne carrément pas pour le transiberien, faut que je fasse une PR sur navitia_model pour rendre le truc mieux, je fais ca tout de suite.

**Comment by antoine-de** [comment]
fix #18
